### ISSUE TITLE
Can't quote ‘tree_filter_cuda’, can you tell me what is the reason please?

### ISSUE THREAD

**Issue by Kobeee0614** [Megvii-BaseDetection/TreeFilter-Torch#3]
Can't quote ‘tree_filter_cuda’, can you tell me what is the reason please?
`ImportError: /home/jason/notebook/TreeFilter-Torch/furnace/kernels/lib_tree_filter/tree_filter_cuda.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZN6caffe26detail36_typeMetaDataInstance_preallocated_7E
`

**Comment by StevenGrove** [comment]
Since the tree filter kernel relies on some components of the PyTorch, please executing "import torch" before "import tree_filter_cuda".

**Comment by Kobeee0614** [comment]
Yes, I imported torch before import tree_filter_cuda, but it seems that this solution is not.`import torch
from torch import nn
import torch.distributed as dist
from torch.autograd import Function
import tree_filter_cuda as _C`

**Comment by StevenGrove** [comment]
This could be caused by the inconsistent version of PyTorch between compiling and runtime. You can try to reinstall PyTorch and refresh the environment, or rerun the installation procedure. In addition, a similar issue has been solved at [link](https://github.com/facebookresearch/maskrcnn-benchmark/issues/119).
